src/libraries/restore/runtime/runtime.depproj

```diff
@@ -74,27 +74,27 @@
   </Target>
 
   <Target Name="OverrideRuntimeCoreCLR"
```

Comment:
If we're unifying the `Resolve*FilesFromLocalBuild` should we unify the `OverrideRuntime*` target as well?

Reply:
I'd prefer doing that in another PR.